sklego/pandas.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import pandas as pd
+from scipy.ndimage.interpolation import shift
+
+
+def _as_list(val):
```

Comment:
If you do `_as_list(range(5))` you get a list where the first element is the generator, is that the behaviour you intend to have? If so, I would suggest updating the docstring as it now mentions that it would return the value as a list if it's not iterable, while in truth it returns the value as a list if not already a list

Reply:
Does this function belong in the `pandas` module or would it make more sense in a utilities module?

Reply:
It is a util for pandas. My attemp with the `.pandas` module was to have utils that only have to do with pandas (this might make sense since the rest of the project is for `sklearn`).

Reply:
"""
    Helper function, always returns a list.

    :param val: a list or a single value.
    :returns: the input value as a list.

    :Example:

    >>> _as_list('test')
    ['test']

    >>> _as_list(['test1', 'test2'])
    ['test1', 'test2']
    """